src/registryctl/api/base.go

```diff
@@ -16,24 +16,36 @@ package api
 
 import (
 	"encoding/json"
+	"github.com/goharbor/harbor/src/lib/errors"
+	server_error "github.com/goharbor/harbor/src/server/error"
```

Comment:
I wish we can clarify the package `harbor/src/server` because that package seems to be for `core` only

Adding dependency from registryctl to sub-package under src/server makes the dependency confusing.

Reply:
It can be moved into `src/lib`, and let `core` and `registrtctl` importing it.

But, I won't do it in this PR.

Reply:
Sure, let's make sure it's tracked.